pyproject.toml

```diff
@@ -138,6 +141,7 @@ ignore_missing_imports = true
 # Per-module configuration options
 module = [
     "datasets",  # SEE: https://github.com/huggingface/datasets/issues/3841
+    "docx",  # SEE: https://github.com/python-openxml/python-docx/issues/1432
```

Comment:
Actually, it looks like they are typed: https://github.com/python-openxml/python-docx/blob/master/src/docx/py.typed

Let's add to `pre-commit` `mypy`'s `additional_requirements`, and remove this change here